src/libvirt-xml-parse.js

```diff
@@ -427,6 +428,17 @@ export function parseDumpxmlForDisks(devicesElem) {
                 readonly: !!readonlyElem,
                 shareable: !!shareableElem,
                 removable: targetElem.getAttribute('removable'),
+                address: { // optional
```

Comment:
This commit should not stand on its own. It adds this new field which is not used anywhere, nor does the commit message justify it. I suppose it should be squashed into some future commit that uses it?